osbs/api.py

```diff
@@ -475,7 +475,6 @@ def _do_create_prod_build(self, git_uri, git_ref,
             koji_kerberos_keytab=self.build_conf.get_koji_kerberos_keytab(),
             koji_kerberos_principal=self.build_conf.get_koji_kerberos_principal(),
             architecture=architecture,
-            platform=platform,
```

Comment:
I see the confusion here. From the diff it looks like `platform` is being removed from the parameter list to `_do_create_prod_build`. Clearly, this is not the case. This looks fine.